certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -193,6 +198,8 @@ def choose_vhost(self, target_name):
         :rtype: :class:`~certbot_nginx.obj.VirtualHost`
 
         """
+        if self.vhost is not None:
```

Comment:
The goal is to make sure that we're modifying the same block when we deploy the cert and when we add ocsp.